src/views/v2/tickets/item/extra/available.php

```diff
@@ -24,9 +24,30 @@
 	return;
 }
 
+$available_count = $ticket->available();
+
+/**
+ * Allows hiding of "unlimited" to be toggled on/off conditionally.
+ *
+ * @since 4.11.1
+ *
+ * @var bool $show_unlimited    Whether to show the "unlimited" text.
+ * @var int  $available_count The quantity of Available tickets based on the Attendees number.
+ */
+$show_unlimited = apply_filters( 'tribe_tickets_block_show_unlimited_availability', true, $available_count );
+
+$context = [
+	'ticket'         => $ticket,
+	'show_unlimited' => (bool) $show_unlimited,
+	'available_count' => $available_count,
+	'is_unlimited'   => - 1 === $available_count,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 4 space(s) between "'is_unlimited'" and double arrow, but found 3.

<sub>hash: 29bd85e9a9e5a92a3bd6e0f9d754d9c8</sub>

Reply:
Let's abstract this into the ticket object itself.